swatch-producer-aws/src/main/java/com/redhat/swatch/aws/processors/BillableUsageAggregateProcessor.java

```diff
@@ -255,7 +252,7 @@ public BatchMeterUsageResponse send(
 
   private UsageRecord transformToAwsUsage(
       AwsUsageContext context, BillableUsageAggregate billableUsageAggregate, Metric metric)
-      throws AwsDimensionNotConfiguredException, UsageTimestampOutOfBoundsException {
+      throws UsageTimestampOutOfBoundsException {
```

Comment:
I'm curious as to why the `AwsDimensionNotConfiguredException` is no longer required since it is not referencing the `uom`?

Reply:
This code should have been removed when we moved the way we used to read tagprofile. 
```
.dimension(
            tagProfile.getAwsDimension(tallySnapshot.getProductId(), measurement.getUom().name())
```